lib/cli.rb

```diff
@@ -25,13 +25,16 @@ def game_loop
     if @game.winner
       display_board
       @output.puts "Congrats #{@game.winner}"
+    elsif @game.board_full?
+      display_board
+      @output.puts "The game is a draw"
     else
       game_loop
     end
   end
 
   def display_board
-    @output.puts @game.board.each_slice(3).map { |s| s.join(' | ') }.join("\n- + - + -\n")
+    @output.puts @game.board_string
```

Comment:
Glad you found this, it's a little more conventional in ruby to override the `to_s` method and then you can just print the board like any other object and it'll be turned into the expected string